### PR TITLE
cat-1318 update cron jobs

### PR DESCRIPTION
**Pull request by kkonuganti**
- update cron jobs to be cleared every 4 days.
- update cron job to use latest k8 version

### CODE REVIEW

File: database_backup/02-cronjob.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
```
Comment (kkonuganti) [inline database_backup/02-cronjob.yaml:2]: this change is to pick up latest k8 version to be used.

File: database_backup/02-cronjob.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ spec:
   failedJobsHistoryLimit: 3
```
Comment (kkonuganti) [inline database_backup/02-cronjob.yaml:10]: this job is run 1st day of every, so setting the job to be cleaned after 4 days of running.

File: database_refresh/02-cronjob.yaml [hunk 3]
```diff
@@ -10,6 +10,7 @@ spec:
   failedJobsHistoryLimit: 3
   jobTemplate:
```
Comment (kkonuganti) [inline database_refresh/02-cronjob.yaml:11]: this batch is run every sunday. So 4 days to clean the job should still be viable.

**Review by Olusola-Opencast — APPROVED** [review]
ok

**Review by kberec — APPROVED** [review]

**Pull request merged**